modules/mvt/test/mvt-loader.binary.js

```diff
@@ -0,0 +1,33 @@
+import test from 'tape-promise/tape';
+import {MVTLoader} from '@loaders.gl/mvt';
+import {setLoaderOptions, fetchFile, parse} from '@loaders.gl/core';
+import {geojsonToBinary} from '@loaders.gl/gis';
+
+setLoaderOptions({
```

Comment:
Ditto

Reply:
This file was more for me to just sanity check my development, but I've now moved it into the spec test, as it doesn't hurt to keep it around. The `setLoaderOptions` is present there, but that seems to be the norm for *spec tests